python/ql/src/experimental/dataflow/internal/DataFlowPublic.qll

```diff
@@ -148,6 +150,35 @@ class ParameterNode extends EssaNode {
   override DataFlowCallable getEnclosingCallable() { this.isParameterOf(result, _) }
 }
 
+class ModuleVariableNode extends Node, TModuleVariableNode {
+  Module mod;
+  GlobalVariable var;
+
+  ModuleVariableNode() { this = TModuleVariableNode(mod, var) }
+
+  override Scope getScope() { result = mod }
+
+  override string toString() {
+    result = "ModuleVariableNode for " + var.toString() + " in " + mod.toString()
+  }
+
+  /** Gets the module in which this variable appears. */
+  Module getModule() { result = mod }
+
+  /** Gets the global variable corresponding to this node. */
+  GlobalVariable getVariable() { result = var }
+
+  /** Gets a node that reads this variable. */
+  Node getARead() { result.asCfgNode() = var.getALoad().getAFlowNode() }
+
+  /** Gets an `EssaNode` that corresponds to an assignment of this global variable. */
+  Node getAWrite() {
```

Comment:
```suggestion
  EssaNode getAWrite() {
```

Reply:
resolved in https://github.com/github/codeql/pull/4265/commits/2d3e23ebb04191d17d3ec03f79e8cef8d6d43643#diff-7c292d1ed419fb354b197f2e84de8be5R180